src/main/java/com/google/sps/servlets/MentorApprovalServlet.java

```diff
@@ -44,22 +44,49 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // Get IDs of mentor and approver.
     int mentorId = Utility.tryParseInt(request.getParameter("id"));
-    int approverId = Utility.getUserId();
+    int userId = Utility.getUserId();
     
     // Set default variables to create MentorEvidence object.
     boolean isApprover = false;
     String mentorUsername = "";
+    boolean isApproved = false;
+    boolean isRejected = false;
     String paragraph = "";
 
     if (userService.isUserLoggedIn()) {
       // If user is logged in, update variables.
-      isApprover = checkForApprover(mentorId, approverId);
+      isApprover = checkForApprover(mentorId, userId);
       mentorUsername = Utility.getUsername(mentorId);
-      paragraph = getMentorEvidence(mentorId);
+
+      // Create the MySQL prepared statement.
+      String query = "SELECT * FROM MentorEvidence "
+          + "WHERE mentor_id = " + Integer.toString(mentorId);
+
+      try {
+        // Establish connection to MySQL database.
+        Connection connection = DriverManager.getConnection(
+            Utility.SQL_LOCAL_URL, Utility.SQL_LOCAL_USER, Utility.SQL_LOCAL_PASSWORD);
+        
+        // Create the MySQL SELECT prepared statement.
+        PreparedStatement preparedStatement = connection.prepareStatement(query);
+        ResultSet queryResult = preparedStatement.executeQuery();
```

Comment:
just wondering, why did you take it out of the helper function?

Reply:
Because originally I only needed one value from the table, but now that I need more and of different type, I find it better to just do it here then create a data structure to return all those values.

Reply:
oh alright, thanks for clarifying!